src/site/_data/authorsData.json

```
@@ -1131,5 +1131,11 @@
     "country": "US",
     "twitter": "stubbornella",
     "image": "image/kheDArv5csY6rvQUJDbWRscckLr1/gjyoj6pRzTfjLXfU7589.jpeg"
+  },
+  "danielasy": {
+    "country": "BR",
+    "github": "danielasy",
+    "linkedin": "danielayassuda",
```

The link did not appear in the preview, so I assume this is the correct format:

```suggestion
    "linkedin": "https://www.linkedin.com/in/danielayassuda/",
```

---

I haven't been able to get these to appear in posts. We can try that out and see if it works.

---

Seems it didn't work
But I don't mind it that much, so don't consider this a blocker for merging